pkg/game/final_scoring_test.go

```diff
@@ -0,0 +1,633 @@
+package game
+
+import (
+	"testing"
+
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/deck"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/game/elements"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/game/position"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/stack"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/feature"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/side"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/tiletemplates"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tilesets"
+)
+
+/*
+  All tests contains the board visualization. Each tile is represented by 5x5 ascii characters
+*/
+
+func makeTurn(game *Game, t *testing.T, tilePosition position.Position, rotations uint, meeple elements.MeepleType, featureSide side.Side, featureType feature.Type) {
```

Comment:
7 _positional_ arguments is a lot - it's hard to figure out the meaning of arguments from the call site alone.

Some of these arguments are closely related and could be grouped in a struct to improve readability - for example: `meeple`, `featureSide`, and `featureType` all refer to a meeple placement and are either set to None or to actual values.

With that said, personally, I found the `rotations` argument the most cryptic since at the call site it just looks like a magic number with an unknown meaning. Pre-rotating it in the tileset slice would both be more readable (since it would be easy to see that the number is passed to `.Rotate()` method) and allow you to remove another argument from this function. Alternatively, you could also turn all of the function's arguments into a single struct which would allow to specify the argument's name at the call site though I don't think it would be that beneficial, if the meeple placement arguments get grouped into a struct too.

Reply:
Is it now more acceptable? It may be a little bit better?

Reply:
Definitely better! Lines are a bit lengthy but it's far from the only place in our codebase where that's the case :P